bin/drop_sample_annot.py

```diff
@@ -132,7 +141,7 @@ def final_annot(count_file: Path, ref_annot: Path, out_file: Path):
     args = parser.parse_args()
     SampleAnnotation(
         args.bam,
-        args.sample,
+        args.samples,
         args.strandedness,
         args.single_end,
         args.gtf,
```

Comment:
You are only using the class for writing a CSV file in a constructor. You should have a specific method for that: 1. define the class with its attributes, 2. use a class method to write the CSV file.

In any case, I don't think we need to create a class in this context

Reply:
That's true, I have made it into a function